packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -102,9 +103,15 @@ export function Modal({
   );
 }
 
+export const ModalSubtitle = classed('strong', 'typo-headline mb-2');
+export const ModalText = classed('p', 'typo-callout text-theme-label-tertiary');
```

Comment:
Are these generic for each modal, else it's maybe best to keep it in the specific modal?
Just not sure if it's recurring or not else we'll have to apply it to other modals as well that use this format.

Reply:
I have seen at least 1 more usage, actually let me remove migrate those as well

Reply:
I think they are being used in other places, I can try to migrate those as well after this PR

Reply:
Sure it can be separate :)
Just nice to have everything from one source instead of re-inventing the wheel.